Dockerfile.tarball

```diff
@@ -0,0 +1,31 @@
+FROM cilium/tetragon:latest as tetragon-release
+
+RUN mkdir -p /etc/tetragon/tetragon.conf.d/
+
+FROM scratch as linux-tarball-1
+
+# Copy the systemd service file
+COPY install/linux-tarball/systemd/tetragon.service /usr/lib/systemd/system/
+
+# COPY configurations
+COPY install/linux-tarball/usr /usr
+COPY --from=tetragon-release /etc/tetragon/ /etc/tetragon/
+
+# Our binaries
+COPY --from=tetragon-release /usr/bin/tetragon /usr/local/bin/
+COPY --from=tetragon-release /usr/bin/tetra /usr/local/bin/
+
+# Helper binaries
+COPY --from=tetragon-release /usr/bin/bpftool /usr/local/lib/tetragon/
```

Comment:
do we need to install bpftool? I think it's part of each major distro

Reply:
Yes, @kkourt suggested to ship our bpftool since it is used part of bugtool logic! maybe it has more new features?

Reply:
Yes I think we should ship our bpftool so we endup with same reports and logic with bugtool and bpttool for both k8s containers environments and standalone bare metal ones.

Reply:
The systemd service contains our own env PATH first which I assume should work

Reply:
ok, makes sense